src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobContainerClientBuilderProvider.java

```diff
@@ -30,6 +30,7 @@ public BlobContainerClientBuilder getBlobContainerClientBuilder() {
 
     public BlobContainerClientBuilder getPcqBlobContainerClientBuilder() {
         return new BlobContainerClientBuilder()
+            .httpClient(httpClient)
```

Comment:
Makes sense to rename `httpClient` to `azureHttpClient`

Reply:
it was there in use. Later there will be just 1 http client.